plugin/pcp-ethereum/pcp-ethereum.go

```diff
@@ -0,0 +1,24 @@
+// This package is the native pocket core plugin for ethereum.
+package pcp_ethereum
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+)
+
+/*
+"ExecuteRequest" takes in the raw json string and forwards it to the ethereum port
+ */
+func ExecuteRequest(jsonStr []byte, ethPort string) string {
```

Comment:
Why not create an util function to forward requests and call it from both plugins?

Reply:
Because we want these plugins to be free of dependencies. I want to set the precedent of plugins being like “Separate packages”